tests/ascii_data_boundary_velocity_residual_shell/original.prm

```diff
@@ -0,0 +1,98 @@
+##### simple test for ascii data
```

Comment:
This file is not helpful as test output, because it will always be the same as the test .prm file. The same is true for the parameters.* files. We usually want to add files into the test ouput directory that will show us if something fails with the functionality of this test, thus `screen-output` and `statistics` are often added. For this test you would want to also add one of the `surface-solution` files to make sure the output of the visualization postprocessor stays the same.